README.adoc

```diff
@@ -1,4 +1,4 @@
-# Red Hat Single Sign-On 7.2 container image
+# Red Hat Single Sign-On container image
```

Comment:
@slaskawi To be consistent, I would propose to use what's in the container catalog:
- https://access.redhat.com/containers/?tab=overview#/registry.access.redhat.com/redhat-sso-7-tech-preview/sso-cd-openshift

Thus `Red Hat Single Sign-On Continuous Delivery (RH-SSO CD) for OpenShift` IMHO.

Reply:
I'm not sure if this is right to be honest. This is a standalone image and it doesn't contain OpenShift related bits (such as `KUBE_PING`). So saying it's for OpenShift is not quite right. 

Let's leave it with minimum set of changes...

Reply:
@slaskawi Right, you are correct with the `OpenShift` piece. Ok, let's keep the current version.